### ISSUE TITLE
[Swap form] was hidden when Metamask display to sign.

### ISSUE THREAD

**Issue by longboynd9** [The-Poolz/Poolz-Back#35]
[Swap form] was hidden when Metamask display to sign.

1. Go to Dashboard: https://app.poolz.finance/dashboard
2. Choose one pool
![image](https://user-images.githubusercontent.com/8565462/103788485-75a1da00-5071-11eb-81fd-c638215e2d0d.png)
3. Put amount to swap.
4. Click Swap function.
5. Metamask extension display to sign. But [Swap Form] was hidden.
![image](https://user-images.githubusercontent.com/8565462/103789179-4a6bba80-5072-11eb-9314-7401f97975cd.png)

Wrong: I think it is not good. Pls review again

Desktop:
OS: Window 10
Browser: Google chorme
Version 85.0.4183.83

Wallet address
0xc7db95F51C81C7A473b3E79cb5E0582FD8873a30

**Comment by Lomet** [comment]
We need to think about it. but it's not a bug.
in case of canceling the wallet operation, the user can re-send it with changes.

**Comment by longboynd9** [comment]
Yep. I see. Personally I think we keep [swap form] is bester than hide it.
